qiskit_experiments/analysis/data_processing.py

```diff
@@ -0,0 +1,104 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+Curve fitting xydata for experiment analysis
+"""
```

Comment:
This file looks more like a collection of utility functions than curve fitting.

Reply:
I need to go over all the doc strings and fix them up after all the reshuffling of where functions are